relentless/potential/potential.py

```diff
@@ -93,12 +88,19 @@ def __init__(self, types, params, default={}):
             raise TypeError('All parameters must be strings')
 
         self.params = tuple(params)
-        self.default = FixedKeyDict(keys=self.params)
-        self.default.update(default)
 
+        # shared params
+        self.shared = FixedKeyDict(keys=self.params)
```

Comment:
Just in case anybody gets any ideas, can you replace this with `self._shared` and make a shared `@property`?